depends/README.md

```diff
@@ -41,7 +41,13 @@ The paths are automatically configured and no other options are needed.
 
 #### Common
 
-    apt install bison cmake curl make patch pkg-config python3 xz-utils
+    apt install cmake curl make patch
+
+#### GUI
+
+Skip the following packages if you don't intend to use the GUI and will build with [`NO_QT=1`](#dependency-options):
+
+    apt install bison pkg-config python3 xz-utils
```

Comment:
pkg-config can only be moved here after #31181?

Reply:
For `make -C depends MULTIPROCESS=1 NO_QT=1` the `pkg-config` seems not required though.

Reply:
It would be more of a convinience. I guess this doesn't make too much difference as 31181 is likely to be merged soon in any case.